tests/Aspire.Azure.Npgsql.EntityFrameworkCore.PostgreSQL.Tests/TokenCredentialTests.cs

```diff
@@ -29,16 +29,17 @@ internal static void ConfigureDbContextOptionsBuilderForTesting(DbContextOptions
     [InlineData(false)]
     public void ReadsUsernameFromToken(bool useEnrich)
     {
+        var connectionString = "Host=localhost;Database=test2;Persist Security Info=True";
```

Comment:
What does `Persist Security Info` do? Was is it needed in these tests?

Reply:
By default, once you call Open() on a DbConnection, the Password should get redacted from the connection string, so if you access DbConnection.ConnectionString, you no longer see it; this is an outdated "security mechanism" to protect against accidental leakage of the password (though of course it's still accessible via reflection...). `Persist Security Info=true` disables this behavior, preserving the password in the connection string.

(no idea why this is used here, just explaining what this does)

Reply:
Kept it only where it was required, there is a test that ensure we are not changing the username and password, so I want to keep the info to compare.